src/openassetio-python/cmodule/src/managerApi/ManagerInterfaceBinding.cpp

```diff
@@ -112,6 +114,48 @@ struct PyManagerInterface : ManagerInterface {
                       successCallback, errorCallback);
   }
 
+  void getWithRelationshipPaged(
+      const EntityReferences& entityReferences, const TraitsDataPtr& relationshipTraitsData,
+      const trait::TraitSet& resultTraitSet, size_t pageSize, const ContextConstPtr& context,
+      const HostSessionPtr& hostSession,
+      const ManagerInterface::PagedRelationshipSuccessCallback& successCallback,
+      const ManagerInterface::BatchElementErrorCallback& errorCallback) override {
+    auto successRetainingWrapper =
+        [&successCallback](
+            size_t idx,
+            const PyRetainingSharedPtr<EntityReferencePagerInterface>& pagerInterface) {
+          successCallback(idx, pagerInterface);
+        };
+
+    PYBIND11_OVERRIDE_IMPL(void, ManagerInterface, "getWithRelationshipPaged", entityReferences,
```

Comment:
[fix] Can we document what's going on here and below as its not using public pybind API and looks like we always call the base class method (which would be v bad).

Reply:
Added and documented a macro to override this in e7293ce1